skqulacs/qnn/regressor.py

```diff
@@ -67,6 +68,20 @@ def __post_init__(self) -> None:
             self.scale_y_scaler = MinMaxScaler(
                 feature_range=(-self.y_norm_range, self.y_norm_range)
             )
+        if self.observables_str != []:
+            # add the right observables
+            for i in self.observables_str:
+                observable = Observable(self.n_qubit)
+                observable.add_operator(1.0, i)
+                self.observables.append(observable)
+        else:
+            # just add Zi for number of i outputs
+            for i in range(self.n_outputs):
```

Comment:
self.n_outputs is initialized 1 vlaue. But it's not so good. line 56
My suggetion is `for i in range(self.n_outputs):` change to `for i in range(1):`

Reply:
Hi, thank you for the comments :) I am trying to resolve this issue. The problem is that sometime we have tests with more observables (2 outputs), so I am trying to make possible to define also 2 observables instead of only 1

Reply:
OK. Can you check `self.n_outputs` is `None`?

Reply:
I may have solved the issue, I will now pr again it seems to work locally now